aana/tests/conftest.py

```diff
@@ -144,43 +150,24 @@ def one_request_worker():
 
 
 @pytest.fixture(scope="function")
-def db_session():
+def db_session(postgresql):
```

Comment:
Now it only tests with Postgres, correct? What about SQLite? We should be running the tests with both of them since we support both.

Reply:
fixed.